src/payment-upon-signature/logic/logic.ergo

```diff
@@ -53,6 +53,10 @@ contract PaymentUponSignature over PaymentUponSignatureContract state PaymentUpo
     return Response{}
   }
 
+  clause danDidACoolThing( request: DanDidACoolThing) : Response {
```

Comment:
Are changes to this template intended? This specific change looks incomplete (and isn't valid Ergo anymore).

Reply:
No -- those are uncommitted changes. I will fix.

Reply:
Removed this change and bumped the version.